pandas/tseries/tests/test_timedeltas.py

```diff
@@ -1513,8 +1513,8 @@ def test_map(self):
 
         f = lambda x: x.days
         result = rng.map(f)
-        exp = np.array([f(x) for x in rng], dtype=np.int64)
```

Comment:
I might show some of these examples in the whatsnew as well (e.g. period/timedelta)